### ISSUE TITLE
403 Forbidden error for grapahana

### ISSUE THREAD

**Issue by menacher** [StackPointCloud/docker-influxdb#6]
The http://yourpublicip returns 403 forbidden from nginx instead of the grapahana dashboard. Is it hosted out of some other port? I am running on mac os with boot2docker. Influx db is working fine and I am able to write and query data from it.

**Comment by ecki** [comment]
Yes this is actualy a problem with a new grafana version. This does not anymore come with the static index file. The grafana-server eneds to be started and then nginx should proxy to port 3000 (or not be used).
